diff --git a/dist/src/vitest-test-runner.js b/dist/src/vitest-test-runner.js
index 29b77daf30e604887406485c70d930ebbe2db229..3d8d70369a8f79c011af818745b71c64a159b6dc 100644
--- a/dist/src/vitest-test-runner.js
+++ b/dist/src/vitest-test-runner.js
@@ -34,14 +34,14 @@ export class VitestTestRunner {
         this.ctx = await vitestWrapper.createVitest('test', {
             config: this.options.vitest?.configFile,
             // @ts-expect-error threads got renamed to "pool: threads" in vitest 1.0.0
-            threads: true,
-            pool: 'threads',
+            threads: false,
+            pool: 'forks',
             coverage: { enabled: false },
             poolOptions: {
                 // Since vitest 1.0.0
-                threads: {
-                    maxThreads: 1,
-                    minThreads: 1,
+                forks: {
+                    maxForks: 1,
+                    minForks: 1,
                 },
             },
             maxWorkers: 1,
